lib/src/fade.dart

```diff
@@ -84,17 +92,18 @@ class FadeAnimatedTextKit extends AnimatedTextKit {
         );
 
   static List<AnimatedText> _animatedTexts(
-    List<String> text,
-    TextAlign textAlign,
-    TextStyle textStyle,
-    Duration duration,
-  ) =>
+          List<String> text,
+          TextAlign textAlign,
+          TextStyle textStyle,
+          Duration duration,
+          double fadeInEnd,
+          double fadeOutStart) =>
       text
-          .map((_) => FadeAnimatedText(
-                _,
-                textAlign: textAlign,
-                textStyle: textStyle,
-                duration: duration,
-              ))
+          .map((_) => FadeAnimatedText(_,
+              textAlign: textAlign,
+              textStyle: textStyle,
+              duration: duration,
+              fadeInEnd: fadeInEnd,
+              fadeOutStart: fadeOutStart))
```

Comment:
Again, the trailing comma should be used so that indentation doesn't impact lines that weren't changed.

Reply:
Done